crates/client/l1-messages/src/error.rs

```diff
@@ -0,0 +1,37 @@
+use mp_felt::Felt252WrapperError;
+
+#[derive(thiserror::Error, Debug, Eq, PartialEq)]
+pub enum L1MessagesConfigError {
+    #[error("File with L1 Messages Worker config not found: `{0}`")]
+    FileNotFound(String),
+    #[error("Failed to deserialize L1 Messages Worker Config from config file: `{0}`")]
+    InvalidFile(String),
```

Comment:
Same. This is a `serde_json::Error`. Keep it as it is given to you.
Use `#[from]` to do conversion

Reply:
I think you can even make those two transparent.